resources/definitions/kingroon_kp3.def.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Kingroon KP3",
```

Comment:
I couldn't find any printer that's just "KP3". On [their website](https://www.kingroon.com/collections/fdm-3d-printer-28) Kingroon shows printers called KP3S and KP3L. Was this one meant to be the KP3L? The dimensions match at least.

Reply:
Their website is under renovation so it is a mess right now. The KP3 was the previous model, it used a bowden style system but it seems they don't sell it anymore. I think the KP3L must be an unreleased prototype, because I've not seen that model anywhere else (the design matches that of the KP3 base, but that one only has one Z arm).